src/common.c

```diff
@@ -445,40 +445,44 @@ get_mentions(gboolean whole_word, gboolean case_sensitive, const char* const mes
 gboolean
 call_external(gchar** argv, gchar** std_out, gchar** std_err)
 {
-    GError *spawn_error, *status_error;
-    gboolean spawn_result;
-    gint exit_status;
+    GError* spawn_error = NULL;
+    GError* exit_error = NULL;
+    gboolean is_successful;
```

Comment:
this is a little too generic for my taste. If you don't like `spawn_result` what do you think about something like `spawn_succesful`?

Reply:
> this is a little too generic for my taste. If you don't like `spawn_result` what do you think about something like `spawn_succesful`?

I reused the same variable for whether the spawn was successful and also whether the exit status was OK or not.

Otherwise we'd introduce two boolean variables `spawn_successful` and `exited_successfully` respectively with an `&&` at the return statement for clarity.

Or just rework the code further making it explicitly return in places like:

```c
if (spawn_successful) {
    if (g_check_spawn_wait_status(...)) {
        return TRUE;
    } else {
        ...;
        return FALSE;
    }
} else {
    ...;
    return FALSE;
}
```

I don't know honestly. This seems like over-thinking to me as well.

Reply:
yeah true its not a big deal. you can also leave it.